cli/internal/fs/turbo_json.go

```diff
@@ -21,8 +21,6 @@ const (
 	topologicalPipelineDelimiter = "^"
 )
 
-var defaultOutputs = TaskOutputs{Inclusions: []string{"dist/**/*", "build/**/*"}}
```

Comment:
I'm like 99.999% confident that `dist/**/*` and `dist/**` are equivalent in doublestar, but I'd love to be 100% sure. Can you triple-check that for us?

Reply:
They are